### PR TITLE
Rra/ivanov/kdoc2klib

### PR DESCRIPTION
**Pull request by knebekaizer**

### CODE REVIEW

**Comment by knebekaizer** [comment]
KT-38600 Kotlin MP iOS Target does not contain kdoc comments

File: compiler/ir/serialization.common/src/org/jetbrains/kotlin/backend/common/serialization/metadata/KlibMetadataSerializerExtension.kt [hunk 1]
```diff
@@ -68,9 +76,28 @@ class KlibMetadataSerializerExtension(
         childSerializer: DescriptorSerializer
     ) {
         descriptorFileId(descriptor)?.let { proto.setExtension(KlibMetadataProtoBuf.functionFile, it) }
+        descriptor.findKDoc()?.let { proto.setExtension(KlibMetadataProtoBuf.functionKdoc, it.render()) }
```
Comment (SvyatoslavScherbina) [inline compiler/ir/serialization.common/src/org/jetbrains/kotlin/backend/common/serialization/metadata/KlibMetadataSerializerExtension.kt:79]: IIUC, (secondary) constructors aren't supported (here and in other parts of the machinery).

**Review by alexander-gorshenev — APPROVED** [review]

File: native/commonizer/src/org/jetbrains/kotlin/descriptors/commonizer/konan/NativeDistributionCommonizer.kt [hunk 2]
```diff
@@ -127,20 +127,20 @@ class NativeDistributionCommonizer(
...
-            resultsConsumer = NativeDistributionResultsConsumer(
-                repository = repository,
-                originalLibraries = allLibraries,
-                destination = destination,
-                copyStdlib = copyStdlib,
-                copyEndorsedLibs = copyEndorsedLibs,
-                logProgress = ::logProgress
-            )
-            dependeeModulesProvider = NativeDistributionModulesProvider.forStandardLibrary(storageManager, allLibraries.stdlib)
+        resultsConsumer = NativeDistributionResultsConsumer(
```
Comment (SvyatoslavScherbina) [inline native/commonizer/src/org/jetbrains/kotlin/descriptors/commonizer/konan/NativeDistributionCommonizer.kt:130]: Is the indent correct here?

Comment (knebekaizer) [reply native/commonizer/src/org/jetbrains/kotlin/descriptors/commonizer/konan/NativeDistributionCommonizer.kt]: Sorry. Rebase is happened to be broken.. This file is not supposed to be changed at all.

**Review by SvyatoslavScherbina — APPROVED** [review]

**Review by SvyatoslavScherbina — APPROVED** [review]

**Pull request merged**